src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.heading1 {
+  font-size: 17px;
+  font-weight: bold;
+}
+
+.heading2 {
+  font-size: 15px;
+  font-weight: normal;
+}
+
+h1,
+p {
+  margin: 0;
+}
+
+h2 {
+  margin: 1em 0;
+}
+
+a {
+  color: green;
+}
```

Comment:
Don't use tags for styling the document.

Add these styles to existing classes, or create new ones if necessary.